docker/onadata-uwsgi/Dockerfile.ubuntu

```diff
@@ -1,110 +1,102 @@
-FROM python:3.10 as intermediate
+FROM onaio/python-deps:3.10.14 as base
 
+ARG release_version=v4.0.1
```

Comment:
@ukanga Are we using this arg?

Reply:
I don't think we are, and for some reason, I have been unwilling to remove it. I suppose there is a workflow where that may make sense but I have yet to try and make it work.